src/app/article.ts

```diff
@@ -0,0 +1,15 @@
+import { Category } from "./category"
+import { User } from "./user"
+
+export interface Article {
+
+    id: number
+    title: string
```

Comment:
add slug parameter

Reply:
Slug will be intergrated will using the datas from US-Interfaces